test/unit_tests/modules/classes/BreakpointHandler-spec.js

```diff
@@ -10,8 +10,8 @@ var BASE_LOC = '../../../../cfgov/unprocessed/js/';
 
 beforeEach( function() {
   args = {
-    enter:      function() {},
-    leave:      function() {},
+    enter:      function() { return; },
```

Comment:
No, it's http://eslint.org/docs/rules/no-empty-function 
Basically it can be unclear whether empty functions are intentional or mistakenly left in the codebase so the rule is about leaving a code comment in the function. In this case, the functions are placeholders that are intended to be overridden. I could include a comment, but then would need to unwrap the `{ }` across multiple lines. It seemed clear that these are placeholders, so I included the `return;` to give them a body (which is implicit anyway, so it shouldn't change behavior).